unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWFloorHandler.cs

```diff
@@ -92,6 +92,7 @@ public void CreateFloor(CatalogItem floorSceneObject)
         {
             DCLBuilderInWorldEntity decentralandEntity = biwCreatorController.CreateCatalogItem(floorSceneObject, WorldStateUtils.ConvertPointInSceneToUnityPosition(initialPosition, parcel), false, true);
             decentralandEntity.rootEntity.OnShapeUpdated += OnFloorLoaded;
+            dclBuilderMeshLoadIndicatorController.HideAllIndicators();
```

Comment:
This won't work in a scene with severals parcels since it will hide the loading indicators of the previous instanciated placeholders. 

I think a better approach will be to delete the complete loading placeholder when deleting the previous floor entities, this way we are not only hiding the indicators but also removing the placeholder that will stay in the world

Reply:
You're right, thanks for the warning! I have changed the approach, you can see it in https://github.com/decentraland/unity-renderer/pull/358/commits/d4125b1ce8409eb8166fce5e8b46d716a4fe546b